airbyte-integrations/bases/base-typing-deduping/src/test/java/io/airbyte/integrations/base/destination/typing_deduping/TypeAndDedupeOperationValveTest.java

```diff
@@ -54,46 +54,48 @@ public void testReadyToTypeAndDedupe() {
     elapseTime(minuteUpdates, 1);
     Assertions.assertTrue(valve.readyToTypeAndDedupe(STREAM_B));
     valve.updateTimeAndIncreaseInterval(STREAM_A);
-    Assertions.assertEquals(1000 * 60 * 5,
+    Assertions.assertEquals(1000 * 60 * 15,
         valve.getIncrementInterval(STREAM_A));
     // method call increments time
     Assertions.assertFalse(valve.readyToTypeAndDedupe(STREAM_A));
-    elapseTime(minuteUpdates, 5);
     // More than enough time has passed now
+    elapseTime(minuteUpdates, 15);
     Assertions.assertTrue(valve.readyToTypeAndDedupe(STREAM_A));
   }
 
   @Test
   public void testIncrementInterval() {
     final var valve = new TypeAndDedupeOperationValve(ALWAYS_ZERO);
     valve.addStream(STREAM_A);
-    IntStream.rangeClosed(1, 3).forEach(i -> {
+    IntStream.rangeClosed(1, 4).forEach(i -> {
       final var index = valve.incrementInterval(STREAM_A);
       Assertions.assertEquals(i, index);
     });
-    Assertions.assertEquals(3, valve.incrementInterval(STREAM_A));
+    Assertions.assertEquals(4, valve.incrementInterval(STREAM_A));
     // Twice to be sure
-    Assertions.assertEquals(3, valve.incrementInterval(STREAM_A));
+    Assertions.assertEquals(4, valve.incrementInterval(STREAM_A));
   }
```

Comment:
@jbfbell I'll be honest... I don't really understand what this test is doing.  But, I added one more increment of time, so the numbers all +1 seems right?

Reply:
ah probably should add a comment explaining - its testing that as you increment the interval, its at that expected interval, (if you're at 1 minute, when you increment you're now at 15 minutes) but with a cap so if you increment when you're already at the last index, you stay there.

But yes your change is correct